Solobarine.github.io/style.css

```diff
@@ -831,4 +842,63 @@ header {
     justify-content: center;
     margin: 25px 0;
   }
+
+  .pop-menu {
+    width: auto;
+    background: #1c1a19;
+    margin-top: 0;
+    padding-top: 0;
+  }
+
+  .pop {
+    display: none;
+  }
+
+  #close {
+    float: right;
+    margin-top: 10px;
+    margin-right: 10px;
+    width: 15px;
+    height: 15px;
+  }
+
+  .img-button {
+    width: 100%;
+    height: 100%;
+  }
+
+  .nav-tools {
+    font-family: 'Inter', sans-serif;
+    font-size: 22px;
+    line-height: 44px;
+    color: #fff5e1;
+    list-style: none;
+    width: 270px;
+    margin-top: 10px;
+    border-bottom: 1px solid #ccc;
+    overflow-y: hidden;
+  }
```

Comment:
Great job team 🤝

- [ ] Just like the previous reviewer pointed out, please try to follow the design provided in the Figma template. You may kindly adjust the color of your mobile popup so that it matches the Figma design.
- [ ] You may also make the closing button the same as the one in the Figma design.
- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/180595469-44b756fb-a5f1-475b-bf56-60c6b502dfae.PNG)

- Your design
![Your design](https://user-images.githubusercontent.com/63917725/180595489-7f2cbe11-f58a-4d68-b2be-7fdddcd07f40.PNG)